docs/sidebars.js

```diff
@@ -0,0 +1,133 @@
+module.exports = {
+  docs: [
+    {
+      type: "doc",
+      id: "home",
+    },
+    {
+      type: 'link',
+      label: 'About Systers',
+      href: 'http://systers.io/'
+    },
+    {
+      type: "doc",
+      id: "Getting-Started",
+    },
+    {
+      type: "category",
+      label: "How-to-Contribute",
+      items: [
+        {
+          type: "doc",
+          id: "How-to-Contribute",
+        },
```

Comment:
`How to Contribute` is twice here could you link  the How-to-Contribute.md doc with the one with type: "category"